ampd/src/commands/register_chain_support.rs

```diff
@@ -18,6 +18,9 @@ pub struct Args {
 }
```

Comment:
how about using a `nonempty::Vec` here instead?

Reply:
this was a bit difficult, we need to derive Valuable on nonempty::Vec, which cascades to a lot of changes across the codebase